.gitignore

```diff
@@ -63,3 +63,6 @@ typings/
 
 # next.js build output
 .next
+
+# VS Code configuration
+.vscode
```

Comment:
We shoud remove this because the config related to editors should be in the personal ~/.gitignore file not in the project .gitignore file.

Reply:
Agreed. I would then clean up `.gitignore` a little more, also removing parts that are not relevant to this project.